src/pages/library/DataBrowserPreview.js

```diff
@@ -49,6 +50,19 @@ const DataBrowserPreview = ({ id }) => {
         table: value
       })
       setPreviewData(previewTableData?.result || [])
+
+      setColumnSettings(
+        _.flow([
+          _.toPairs,
+          _.map(([index, col]) => {
+            return {
+              name: col.name, key: col.name,
+              visible: index < 6,
```

Comment:
I'm curious about the 6 index.  This feels like it could break if more columns are added early on.  It's probably worth a comment or maybe look up the column where things start showing up (assuming that the 6th column is meaningful).  Also, what happens if a user re-orders columns?  Just not sure what the rational is for hiding all columns that are at higher indexes

Reply:
I picked a random index, just because it would've looked weird with more columns than that, the table gets really squished! There is a column selector that allows the user to reorder the columns and choose the visibility as well, so the 6 column limit is an initial setting, totally adjustable. Since the visibility field is tied with the columns, if they reorder them, they wont disappear (if the new index  > 6)

Reply:
Ah ok, I gotcha

Reply:
It'd be great to avoid [magic numbers](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544) like `6` above :)